### ISSUE TITLE
Assertion failure - build with "-enable-debug"

### ISSUE THREAD

**Issue by janekptacijarabaci** [MoonchildProductions/UXP#631]
After depends on:
#630 

```
Assertion failure: iEV < NUM_TEST_EV_ROOTS (Could not find built-in EV root),
at [path]/UXP/security/certverifier/ExtendedValidation.cpp:1295
```

Ad:
https://bugzilla.mozilla.org/show_bug.cgi?id=1227638
```
-      if (iEV < NUM_TEST_EV_ROOTS) {
-        continue;
+      // The entries for the debug EV roots are at indices 0 through
+      // NUM_TEST_EV_ROOTS - 1. Since they're not built-in, they probably
+      // haven't been loaded yet.
+      MOZ_ASSERT(iEV < NUM_TEST_EV_ROOTS, "Could not find built-in EV root");
```

Does it make sense to PM/Basilisk/UXP?

**Comment by janekptacijarabaci** [comment]
The debug mode can not be used (and start the browser) without fix this problem.

**Comment by wolfbeast** [comment]
Solved by c534a0c
